api/db/migrations/20190620160455_add_isMigratedToV2_to_user.js

```diff
@@ -0,0 +1,14 @@
+const TABLE_NAME = 'users';
+const COLUMN_NAME = 'isMigratedToV2';
```

Comment:
Point de vue ontologique `is` ne va pas très bien avec `migrated`, -->`hasMigrated`. Absolument non bloquant je ne veux même pas le changer.